token/core/zkatdlog/crypto/audit/auditor.go

```diff
@@ -280,7 +280,7 @@ func inspectTokenOwnerOfScript(des Deserializer, token *AuditableToken, index in
 	}
 	scriptInf := &interop.ScriptInfo{}
```

Comment:
Shall we call ScriptInfo HTLCInfo?

Reply:
this could be a generic script and not just HTLC